src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/SignatureHelp/SignatureHelpEndpoint.cs

```diff
@@ -15,16 +13,14 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.SignatureHelp;
 
-using SignatureHelp = RLSP::Roslyn.LanguageServer.Protocol.SignatureHelp;
-
 [RazorLanguageServerEndpoint(Methods.TextDocumentSignatureHelpName)]
 internal sealed class SignatureHelpEndpoint(
         LanguageServerFeatureOptions languageServerFeatureOptions,
         IDocumentMappingService documentMappingService,
         IClientConnection clientConnection,
         RazorLSPOptionsMonitor optionsMonitor,
         ILoggerFactory loggerProvider)
-    : AbstractRazorDelegatingEndpoint<SignatureHelpParams, SignatureHelp?>(
+    : AbstractRazorDelegatingEndpoint<SignatureHelpParams, LspSignatureHelp?>(
```

Comment:
interesting, I wouldn't have expected the type name to change here since it was `Roslyn.LanguageServer.Protocol.SignatureHelp`

Reply:
The type name didn't change. I updated it to use the alias defined in our GlobalUsings.cs:

https://github.com/dotnet/razor/blob/5f20402bbe510c91e4ec82667de9ca6274592b11/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/GlobalUsings.cs#L20

 We're inside a namespace called "SignatureHelp" and using the `LspSignatureHelp` alias avoids needing the SigntureHelp using type alias directive on line 18 above.

Reply:
Ah perfect 👍